state/executor.go

```diff
@@ -591,6 +591,26 @@ func (t *Transition) GetBalance(addr types.Address) *big.Int {
 	return t.state.GetBalance(addr)
 }
 
+func (t *Transition) AddBalance(addr types.Address, balance *big.Int) {
+	t.state.AddBalance(addr, balance)
+}
+
+func (t *Transition) SubBalance(addr types.Address, balance *big.Int) {
+	t.state.SubBalance(addr, balance)
+}
+
+func (t *Transition) GetStakedBalance(addr types.Address) *big.Int {
+	return t.state.GetStakedBalance(addr)
+}
+
+func (t *Transition) AddStakedBalance(addr types.Address, balance *big.Int) {
+	t.state.AddStakedBalance(addr, balance)
+}
+
+func (t *Transition) SubStakedBalance(addr types.Address, balance *big.Int) {
+	t.state.SubStakedBalance(addr, balance)
+}
```

Comment:
I believe we use the rest of the unused StakedBalance functions in the unstaking PR (#76)

Reply:
Then they should be a part of that PR. 🙂

Reply:
@zivkovicmilos Are they all used there? Also getters and subBalance?

Reply:
I think so. 
I'll go ahead and remove them from this branch, since they are present and used on the other one. You're right - it shouldn't be a part of this PR 💯

Reply:
@zivkovicmilos Be careful there!
If you remove them here and later on merge this branch into PoS one, when the time comes to merge the Unstaking branch with the PoS you will pull changes from the PoS branch which will remove these functions from the unstaking branch as well, so take note to add them back there.

Reply:
Removed 👍

Reply:
@zivkovicmilos You can remove `AddBalance` and `SubBalance` that you've added to the executor. They're unused as well.

Reply:
Removed 👍